specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-10-01/examples/UsageDetailsListByMCABillingAccount.json

```diff
@@ -0,0 +1,85 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01",
+    "scope": "providers/Microsoft.Billing/BillingAccounts/1234:56789"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/BillingAccounts/1234:56789/providers/Microsoft.Consumption/usageDetails/usageDetails_Id1",
+            "name": "usageDetails_Id1",
+            "type": "Microsoft.Consumption/usageDetails",
+            "kind": "modern",
+            "tags": {
+              "env": "newcrp",
+              "dev": "tools"
+            },
+            "properties": {
+              "billingAccountId": "1234:56789",
+              "billingAccountName": "Account Name 1",
+              "billingPeriodStartDate": "2021-10-01T00:00:00.0000000Z",
+              "billingPeriodEndDate": "2019-10-31T00:00:00.0000000Z",
+              "billingProfileId": "2468",
+              "billingProfileName": "Account Name 1",
+              "subscriptionGuid": "00000000-0000-0000-0000-000000000000",
+              "subscriptionName": "Subscription Name 1",
+              "date": "2019-10-30T00:00:00.0000000Z",
+              "meterId": "00000000-0000-0000-0000-000000000000",
+              "quantity": 0.7329,
+              "unitPrice": 4.38,
+              "billingCurrencyCode": "USD",
+              "resourceLocation": "USEast",
+              "consumedService": "Microsoft.Storage",
+              "instanceName": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/Resource Group 1/providers/Microsoft.Storage/storageAccounts/Resource Name 1",
+              "invoiceSectionId": "98765",
+              "invoiceSectionName": "Invoice Section 1",
+              "costCenter": "DEV",
+              "resourceGroup": "Resource Group 1",
+              "isAzureCreditEligible": false,
+              "chargeType": "Usage",
+              "additionalInfo": "{  \"UsageType\": \"ComputeHR\",  \"ImageType\": \"Windows Client BYOL\",  \"ServiceType\": \"Standard_D1\",  \"VMName\": null,  \"VMProperties\": null,  \"VCPUs\": 1,  \"CPUs\": 0}",
+              "costInBillingCurrency": 1.84763819095477,
+              "costInPricingCurrency": 1.84763819095477,
+              "exchangeRate": "1",
+              "exchangeRateDate": "2021-10-01T00:00:00Z",
+              "invoiceId": "",
+              "previousInvoiceId": "",
+              "pricingCurrencyCode": "USD",
+              "product": "Virtual Machines D Series - D1 - US East",
+              "productIdentifier": "DZH318Z0BQ4B00FV",
+              "productOrderId": "a3db7880-70eb-4b4c-6a79-1425a058df5a",
+              "productOrderName": "Azure plan",
+              "publisherName": "Microsoft",
+              "publisherType": "1PP",
```

Comment:
@mitagarg This should not be 1pp. Should be either Microsoft or AWS.

@shrutis06 can you confirm?

Reply:
@adwise3 updated